src/main/java/com/partymusicapp/Controllers/SongControllerImpl.java

```diff
@@ -11,8 +11,8 @@ public class SongControllerImpl {
     SongService songService;
 
     @PutMapping(value = "/add", consumes = MediaType.APPLICATION_JSON_VALUE)
-    void addSong (@PathVariable String partyId, @RequestBody Song song){
-        songService.addSong(partyId, song);
+    Song addSong (@PathVariable String partyId, @RequestBody Song song){
```

Comment:
Poti modifica implementarile la controller si service sa nu returneze Object, ci ResponseEntity<Object>. De exemple aici va fi ResponseEntity<Song> si in metoda de la service ii dai return ResponseEntity.ok(song)

Reply:
Asta faci numa daca mai ai timp, mai bine rezolvi celalalte probleme